### PR TITLE
[Programmes 6832] Fixed missing image issue

### PR DESCRIPTION
**Pull request by tomwinnington**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6832

Link:
https://www.bbc.co.uk/programmes/b006q2x0/articles

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsShared/Utilities/Image/ImagePresenter.php [hunk 1]
```diff
@@ -56,6 +56,12 @@ public function __construct(
         $this->sizes = $this->buildSizes($sizes);
     }
 
+    public function hasImage(){
+        if ($this->imagePid !==''){
```
Comment (BjornTwachtmann) [inline src/DsShared/Utilities/Image/ImagePresenter.php]: Is there ever a case where an empty image PID is passed in here? That seems like more of an error in the calling class to me than something we should be checking for here? Or is this a valid use case?

File: src/DsShared/Utilities/Image/image.html.twig [hunk 2]
```diff
@@ -1,5 +1,11 @@
+{% if image.hasImage() %}
+    {% set Srcsets = image.getSrcsets() %}
```
Comment (Tasiobg) [inline src/DsShared/Utilities/Image/image.html.twig]: In PHP we use "camel case" for variables, not sure if this really matter for twig but could look more consisten with "camel case"
{% set srcsets = image.getSrcsets() %}
```suggestion
    {% set srcsets = image.getSrcsets() %}
```

**Comment by Tasiobg** [comment]
Some information I'm not sure if could affect this ticket or PR, I think Images PIDs from iSite use the LIVE images not the ones displayed on test iSite

**Pull request merged**